content/06.methods.md

```diff
@@ -339,6 +339,7 @@ High-grade glioma (HGG) subtypes were derived (`molecular-subtyping-HGG` analysi
 6. All other high-grade glioma samples that did not meet any of these criteria were subtyped as `HGG, H3 wildtype`.
 
 Embryonal tumors were included in non-MB and non-ATRT embryonal tumor subtyping (`molecular-subtyping-embryonal` analysis module) if they met any of the following criteria:
+
```

Comment:
This change and the next 2 are to make sure these criteria render as numbered lists.